common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -232,6 +234,7 @@ public static ObjectMapper objectMapper() {
                   .configure(EnumFeature.WRITE_ENUMS_TO_LOWERCASE, true)
                   .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS)
                   .build()
+                  .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
```

Comment:
Why do we need this?

Reply:
Because when writing and reading MySQL data, when `serializing` and `deserializing` object such as `AuditInfo`, `SchemaVersion`, `Jackson` needs to use the `Getter` and `Setter` methods of the corresponding class. These are not currently provided. I'm not sure the rationale for adding the `Getter` and `Setter` method to it. Therefore, I expanded the Mapper's visible domain for attributes.

Reply:
Let me try to create a new ObjectMapper